crates/ruff_linter/src/rules/flake8_pyi/rules/future_annotations_in_stub.rs

```diff
@@ -10,9 +10,10 @@ use crate::checkers::ast::Checker;
 /// statement in stub files.
 ///
 /// ## Why is this bad?
-/// Stub files are already evaluated under `annotations` semantics. As such,
-/// the `from __future__ import annotations` import statement has no effect
-/// and should be omitted.
+/// Stub files natively support forward references in all contexts, as stubs are
+/// never executed at runtime. (They should be thought of as "data files" for
+/// type checkers.) As such, the `from __future__ import annotations` import
```

Comment:
This might be correct but it's weird that the `.` is inside of the parentheses :sweat_smile: 

```suggestion
/// type checkers). As such, the `from __future__ import annotations` import
```

Reply:
The rule is that if it's a complete sentence inside the parentheses, the period goes inside the parentheses, but if it's only part of a sentence inside the parentheses, the period goes outside: https://www.masterclass.com/articles/period-inside-or-outside-parentheses